app/assets/stylesheets/base.scss

```diff
@@ -65,16 +65,18 @@ pre { word-wrap: break-word; }
 
 .back-to-top {
   background-color: $border-dark-grey;
-  border-radius: 10px;
+  border-radius: 4px;
   bottom: 20px;
   color: $white;
   display: block;
   font-size: 2.9em;
+  height: 50px;
   line-height: 1.5;
   opacity: 0;
-  padding: 0 12px;
   position: fixed;
   right: 54px;
+  transition: opacity ease 400ms;
+  width: 50px;
```

Comment:
@svbergerem pronto asked me to place the width there, feels weird to me, I use to place it next to height... Any reason it should be here?